README.md

```diff
@@ -8,32 +8,102 @@
 
 # PWA Expert System
 
-The goal is to build state transition graphs, going from an initial state to a
-final state. A state transition graph consists of nodes and edges/lines (in
-correspondence to Feynman graphs):
-
-- We call the connection lines **particle lines**. These are basically a list
-  of quantum numbers (QN) that define the particle state. (This list can be
-  empty at first).
-- The nodes are of type `InteractionNode` and contain all information for the
-  transition of this specific step. An interaction node has 𝑀 ingoing lines
-  and 𝑁 outgoing lines (𝑀, 𝑁 ∈ 𝕫, 𝑀 > 0, 𝑁 > 0).
-
-## Concept of building graphs
-
-### Step 1
-Building of all possible topologies. A **topology** is a graph, in which the
-edges and nodes are empty (no QN information). See the topology sub-modules.
-
-### Step 2
-Filling the topology graphs with QN information. This means initializing the
-topology graphs with the initial and final state QNs and *propagating* these
-numbers through the complete graph. Here, the combinatorics of the initial and
-final state should also be taken into account.
-
-### Step 3
-Duplicate the graphs and insert concrete particles for the edges (inserting the
-mass variable).
-
-### Step 4
-Write output to XML model file.
+## About
+
+The two purposes of the Partial Wave Analysis Expert System are to:
+
+1. validate a particle reaction, based on given information. E.g.: Can a 𝜋⁰
+   decay into 1, 2, 3 𝛾 particles?
+2. create partial wave analysis amplitude models, based on basic information of
+   a reaction. E.g.: Create an amplitude for J/𝜓 → 𝛾𝜋⁰𝜋⁰ in the helicity or
+   canonical formalism.
+
+The user only has to provide a basic information of the particle reaction, such
+as an initial state and a final state. Helper functions provide easy ways to
+configure the system, but the user still has full control. The expert system
+then constructs several hypotheses for what happens during the transition from
+initial to final state. Read more on this in the [Design section](#Design).
+
+## Available features
+
+- [ ] **Input**: Particle database
+  - [ ] Source of truth: PDG
+  - [x] Option to overwrite and append with custom particle definitions
+- [ ] **State transition graph**
+  - [ ] ...
+- [ ] **Conservation rules**
+  - [ ] Easily extendable
+    - [ ] User input
+    - [ ] Open-closed design
+  - [ ] ...
+- [ ] **Generate amplitude models for PWA formalisms**
+  - [x] Helicity formalism
+  - [x] Canonical formalism
+  - [ ] Spin projection formalisms
```

Comment:
So I added these with https://github.com/ComPWA/PWA-pages/pull/4 in mind

Reply:
Ok, whats spin projection formalism? Sounds like a general thing, because helicity and canonical are also doing that...

Reply:
I got it from https://arxiv.org/pdf/hep-ph/0412069.pdf, page 6. But what I put here is indeed a bit vague.

Reply:
7c495222fe3e93e7db951970e52918f42730b214